### ISSUE TITLE
How to reference local libraries?

### ISSUE THREAD

**Issue by mustaqimM** [emacs-lsp/lsp-java#226]
Hi there, how do I reference local jar libraries in non-maven/gradle/spring project like with the setting in vscode: `java.project.referencedLibraries`

I've been looking for it something similar but can't find it

**Comment by yyoncho** [comment]
there is call lsp-register-custom-settings - probably we havent exposed the property.

**Comment by mustaqimM** [comment]
I see in vscode, simply putting the libraries in a `lib` folder allows it to resolve errors

**Comment by mustaqimM** [comment]
Actually it does add the libraries to the classpath. I was having trouble making it import the libs so I closed emacs and renamed the project and then it was able to add the lib folder to the class path. Please close this if you wish

And thank you for this and dap-mode!

**Comment by x-ji** [comment]
Yep, for me it worked after I moved the jar into a `lib` folder and ran `lsp-restart-workspace`.
